packages/server/src/automations/steps/createUser.js

```diff
@@ -16,9 +16,10 @@ module.exports.definition = {
   schema: {
     inputs: {
       properties: {
-        username: {
```

Comment:
should probably request a name for the user as well as an email in the automation now as related to previous comments.

Reply:
@mike12345567 the UI for creating users has never let you enter a name though has it? Just username. Just wondering what are the use cases where we need to know their name. I guess if you wanted an automation email to address them by name or something.

Reply:
Yeah, its not necessarily all that useful, its just that its specified in the codebase, if its not a feature we want to use then could just remove references to it and users can add names if they like since its just a table!

Reply:
Going to remove the `name` as above